packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceChart/Footer/template.js

```diff
@@ -44,21 +42,13 @@ const Footer = ({
   return (
     <Wrapper>
       {isCoinifySupported && isSilverOrAbove && coinName === 'Bitcoin' && (
```

Comment:
Let's get rid of this check since SB supports all coins and potentially all users.